src/lib/codec-wrappers/mozjpeg-enc.ts

```diff
@@ -57,13 +61,13 @@ export class MozJpegEncoder implements Encoder {
     })();
   }
 
-  async encode(data: ImageData): Promise<ArrayBuffer> {
+  async encode(data: ImageData, options: EncodeOptions): Promise<ArrayBuffer> {
     const m = await this.emscriptenModule;
     const api = await this.api;
 
     const p = api.create_buffer(data.width, data.height);
     m.HEAP8.set(data.data, p);
-    api.encode(p, data.width, data.height, 2);
+    api.encode(p, data.width, data.height, options.quality!=null ? options.quality : 7);
```

Comment:
We should throw (or something) if a required option is not given. The UI can define a default, the module itself shouldn’t imo.

Reply:
I'm fine with that, or have it default to 100

Reply:
Throwing if quality is missing seems like the right rule for now. We can revisit this as we add more compression options.

Reply:
Done.